Formula/gdcm.rb

```diff
@@ -0,0 +1,33 @@
+class Gdcm < Formula
+  desc "Grassroots DICOM library and utilities for medical files"
+  homepage "https://sourceforge.net/projects/gdcm/"
+  url "https://downloads.sourceforge.net/project/gdcm/gdcm%202.x/GDCM%202.8.2/gdcm-2.8.2.tar.gz"
+  sha256 "5462c7859e01e5d5d0fb86a19a6c775484a6c44abd8545ea71180d4c41bf0f89"
+  revision 1
+
+  depends_on "cmake" => :build
+  depends_on "openssl"
+  depends_on "vtk"
+
+  needs :cxx11
+
+  def install
+    mkdir "build" do
+      args = std_cmake_args + %w[
+        -DGDCM_BUILD_APPLICATIONS=ON
```

Comment:
what does this do

Reply:
From their wiki:

> This will build all command line executable (gdcminfo, gdcmdump, gdcmtar, gdcmanon ... )

Looks fine to have these around

Reply:
Yeah, I would think we'd want these included. We're not optimizing hard for build time or installation size. Unless they pull in additional dependencies.

Reply:
Just wanted to make sure this didn't do something goofy like build a .app